Core/Object Arts/Dolphin/MVP/Presenters/Text/UI.TextEdit.cls

```diff
@@ -733,7 +733,7 @@ onInputMaskRequired: anEvent
 	WM_GETDLGCODE to ensure the control never actually receives it. This keeps 
 	its behaviour sane, and means we do not need workarounds in WM_CHAR (to 
 	forcibly insert VK_TABs). It also enables us to implement WantTab functionality
-	which windows does not support directly as an Edit control style."
+	which Windows does not support directly as an Edit control style."
```

Comment:
"functionality, which"
(or without the comma, "functionality that", as you prefer)